common/tools/dev-tool/src/commands/run/testNodeTSInput.ts

```diff
@@ -24,7 +24,9 @@ export default leafCommand(commandInfo, async (options) => {
   const isModuleProj = await isModuleProject();
   const reporterArgs =
     "--reporter ../../../common/tools/mocha-multi-reporter.js --reporter-option output=test-results.xml";
-  const defaultMochaArgs = `${reporterArgs} --full-trace`;
+  const defaultMochaArgs = `${
```

Comment:
Hm I think Jeremy had a similar fix in main - could you try bringing in the latest changes from main?